frontend/simple/views/components/Chatroom/ChatMain.vue

```diff
@@ -0,0 +1,328 @@
+<template>
+  <div class="c-chatmain" :class="{ isActive: summary.title }">
+    <template v-if="summary.title">
+      <chat-header
+        :title="summary.title"
+        :description="summary.description"
+        routerBack="/messages"
+      >
+        <template slot="actions">
+          <button class="button is-icon">
+            <i class="fa fa-user-plus"></i>
+          </button>
+          <button class="button is-icon">
+            <i class="fa fa-bell"></i>
+          </button>
+          <menu-parent class="level-right">
+            <menu-trigger class="is-icon">
+              <i class="fa fa-ellipsis-v"></i>
+            </menu-trigger>
+
+            <!-- TODO later - be a drawer on mobile -->
+            <menu-content class="c-actions-content">
+              <list hasMargin>
+                <menu-item tag="button" itemId="hash-1" icon="heart">
+                  Option 1
+                </menu-item>
+                <menu-item tag="button" itemId="hash-2" icon="heart">
+                  Option 2
+                </menu-item>
+                <menu-item tag="button" itemId="hash-3" icon="heart">
+                  Option 3
+                </menu-item>
+              </list>
+            </menu-content>
+          </menu-parent>
+        </template>
+      </chat-header>
+
+      <div class="c-body is-flex" :style="bodyStyles">
+        <div v-if="details.isLoading">
+          LOADING... <!-- TODO Design a cool skeleton loading -->
+        </div>
+        <template v-else>
```

Comment:
Why is a `<template>` introduced here? Shouldn't you be able to put the `v-else` on the `<div>` inside?

Reply:
Yes, you are right, it was needed in the past, now it isn't anymore :)